tlslite/tlsrecordlayer.py

```diff
@@ -912,3 +928,45 @@ def _changeWriteState(self):
 
     def _changeReadState(self):
         self._recordLayer.changeReadState()
+
+
+    def writeHeartbeat(self, type, payload, padding_length):
```

Comment:
given the differences between `write`, `writeAsync`, `read` and `readAsync`, it seems to like the more consistent name for this method would be `writeHeartbeatAsync` and for the method below to be called `writeHeartbeat`